cmake/Findepoxy.cmake

```diff
@@ -0,0 +1,90 @@
+#[=======================================================================[.rst:
+Findepoxy
+---------
+
+Try to find libepoxy on a Unix system.
+
+This will define the following variables:
+
+``epoxy_FOUND``
+    True if (the requested version of) libepoxy is available
+``epoxy_VERSION``
+    The version of libepoxy
+``epoxy_LIBRARIES``
+    This should be passed to target_link_libraries() if the target is not
+    used for linking
+``epoxy_INCLUDE_DIRS``
+    This should be passed to target_include_directories() if the target is not
+    used for linking
+``epoxy_DEFINITIONS``
+    This should be passed to target_compile_options() if the target is not
+    used for linking
+``epoxy_HAS_GLX``
+    True if GLX support is available
+``epoxy_HAS_EGL``
+    True if EGL support is available
+
+If ``epoxy_FOUND`` is TRUE, it will also define the following imported target:
+
+``epoxy::epoxy``
+    The epoxy library
+
+In general we recommend using the imported target, as it is easier to use.
+Bear in mind, however, that if the target is in the link interface of an
+exported library, it must be made available by the package config file.
+#]=======================================================================]
+
+find_package(PkgConfig QUIET)
+pkg_check_modules(PKG_epoxy QUIET epoxy)
+
+set(epoxy_VERSION ${PKG_epoxy_VERSION})
+set(epoxy_DEFINITIONS ${PKG_epoxy_CFLAGS})
+
+find_path(epoxy_INCLUDE_DIRS
+    NAMES epoxy/gl.h
+    HINTS ${PKG_epoxy_INCLUDEDIR} ${PKG_epoxy_INCLUDE_DIRS}
+)
+find_library(epoxy_LIBRARIES
+    NAMES epoxy
+    HINTS ${PKG_epoxy_LIBDIR} ${PKG_epoxy_LIBRARY_DIRS}
+)
```

Comment:
Convention in CMake find modules is that  `_LIBRARIES` is an output variable, not an in/out, FWIW, Ideally this would be finding the `_LIBRARY_RELEASE` and the `_LIBRARY_DEBUG` separately, then using `select_library_configurations()` to swap them as appropriate. But I see that libepoxy is a bad citizen in that is doesn't suffix its debug libraries and uses an inferior build system.